plugin/evm/customheader/dynamic_fee_state.go

```diff
@@ -43,7 +44,17 @@ func feeStateBeforeBlock(
 		}
 	}
 
-	state.AdvanceTime(timestamp - parent.Time)
+	switch {
+	case config.IsGranite(timestamp):
+		parentMS := parent.Time * 1000
+		if config.IsGranite(parent.Time) {
+			parentMS = *customtypes.GetHeaderExtra(parent).TimeMilliseconds
+		}
+		state.AdvanceMilliseconds(timeMS - parentMS)
+	case config.IsFortuna(timestamp):
+		// If the parent block was not running with ACP-176, we start with the
```

Comment:
Start with the what? :)

Reply:
Seems like this comment may have been an accidental copy and should be removed actually